tests/conftest.py

```diff
@@ -2470,6 +2471,40 @@ def finalizer():
     return factory
 
 
+@pytest.fixture(scope='function')
+def couchbase_factory_fixture(request):
+    """
+    Couchbase factory fixture
+    """
+    couchbase = CouchBase()
+
+    def factory(replicas=3):
+        """
+        Factory to start couchbase workload
+
+        Args:
+            replicas (int): Number of couchbase workers to be deployed
+        """
+        # Setup couchbase
+        couchbase.setup_cb()
+        # Create couchbur Dase workers
```

Comment:
Typo here, please correct. It should be couchbase

Reply:
done